lib/src/cli/deployapps/baas_client.dart

```diff
@@ -90,21 +97,16 @@ class BaasClient {
         result[app.name] = app;
       }
     }
-    String appName = "flexible";
-    if (!result.containsKey(appName)) {
-      result[appName] = await _createApp(appName);
-    }
-    appName = "autoConfirm";
-    if (!result.containsKey(appName)) {
-      result[appName] = await _createApp(appName, confirmationType: 'auto');
-    }
+    await _createAppIfNotExists(result, defaultAppName);
+    await _createAppIfNotExists(result, "autoConfirm", "auto");
+    await _createAppIfNotExists(result, "emailConfirm", "email");
```

Comment:
[nit] I think it's better to use named arg for the confirmation type instead of a positional one as it makes it more explicit what exactly is auto/email

Reply:
Done.